build.env

```diff
@@ -41,7 +41,8 @@ VM_DRIVER=none
 CHANGE_MINIKUBE_NONE_USER=true
 
 # Rook options
-ROOK_VERSION=v1.3.9
+ROOK_VERSION=v1.4.9
+#ROOK_CEPH_CLUSTER_VERSION=v14.2.12
```

Comment:
How about having complete image URL here. If we do that we can use custom image from our own dockerhub or quay for our local testing

Reply:
that would be nice, indeed

Reply:
@Madhu-1 Where can I find the image URL? Any example

Reply:
`docker.io/ceph/ceph:v15` would be the URL (not really URL, more of a fully-qualified-container-image-name) for the base image